docs/learn/learn-nft-pallets.md

```diff
@@ -24,7 +24,52 @@ on Westmint, which provides a multitude of functions to interact with NFTs.
 
 The pallet comes with a new way to configure NFTs, as well as configure collections and items.
 Pallet-level [feature flags](https://github.com/paritytech/substrate/pull/12367) allow disabling
-functionalities that are not needed in the runtime.
+functionalities not needed in the runtime.
+
+### Roles
+
+Setting up a collection implies different roles with different permissions:
+
+- Owner:
+
+  - destroy collection (to destroy the collection, there should be 0 items left).
+  - redeposit: re-evaluate the deposit on some items.
+  - set team: change the collection’s Issuer, Admin, Freezer.
+  - set collection max supply: set the maximum number of items for a collection.
+  - lock collection: this can include making a collection’s items non-transferable, fixing its max
+    supply, and locking collection metadata and attributes.
+
+- Admin:
+  - set attribute and metadata of a collection.
+  - set attributes pre-signed: set attributes for an item by providing the pre-signed approval.
```

Comment:
"set attributes for an item by providing the pre-signed approval." -> "set attributes for an item by providing the Admin pre-signed approval."